.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: rust
+rust: 1.34.2
+dist: xenial
+sudo: true
```

Comment:
Why do we need sudo here?

Reply:
Not sure, I just copy the .travis.yml template from CKB